src/Microsoft.AspNetCore.Server.Kestrel/Http/Frame.cs

```diff
@@ -696,8 +696,129 @@ private Task WriteSuffix()
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private unsafe void EmitHttp11(ref MemoryPoolIterator end)
+        {
+            byte* b;
+            
+            if (end.GetRawBuffer(9, out b))
+            {
+                 // Emit directly
+                *(long*)b = 0x312E312F50545448;
```

Comment:
Mostly modern x86/x64 don't care as long as you don't cross a page boundary (when you can have a performance hit). All the memory blocks are page aligned so no writes will cross page boundaries.

x86 doesn't like writing longs though :)